Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env groovy
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  try {
+    stage("Checkout") {
+      checkout scm
+    }
+
+    stage("Cleanup") {
+      govuk.cleanupGit()
+      govuk.mergeMasterBranch()
+    }
+
+    stage("Build") {
+      sh "${WORKSPACE}/jenkins.sh"
+    }
+
+    stage("Publish gem") {
+      if (env.BRANCH_NAME == 'master') {
```

Comment:
Could this be outside of the stage, so that the stage does not appear to be run when building the branches for Pull Requests?